src/components/services/patches.js

```diff
@@ -1,8 +1,9 @@
-export function savePatch(patch) {
-  localStorage.setItem("Patch", JSON.stringify(patch));
+import { useState } from "react";
+
+export function useSavePatch(patch) {
+  const [savedPatches, setSavedPatches] = useState(loadPatch());
 }
 
 export function loadPatch() {
-  const loadedPatch = JSON.parse(localStorage.getItem("Patch"));
   return loadedPatch;
 }
```

Comment:
This file is not used anymore, probably you could remove it or consider writing a custom hook for loading and saving the patches in your `Load`  and `Save` components.

Maybe too big to be solved in this PR.